hi-ml-multimodal/src/health_multimodal/image/inference_engine.py

```diff
@@ -67,3 +68,19 @@ def get_patch_embeddings_from_image(self, image_path: Path) -> Tuple[torch.Tenso
         assert projected_img_emb.shape[0] == 1
 
         return projected_img_emb[0], img_shape
+
+    @torch.no_grad()
+    def get_projected_global_embedding_from_image(self, image_path: Path) -> torch.Tensor:
+        """Compute global image embedding in the joint latent space.
+
+        :param image_path: Path to the image to compute embeddings for.
+        :return: Torch tensor containing l2-normalised global image embedding [joint_feature_dim,]
```

Comment:
Maybe it can be made a bit more explicit what `[joint_feature_dim,]` means here.

Reply:
updated.